src/api/index.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios'
+
+const api = axios.create({
+  baseURL: import.meta.env.REACT_APP_API_URL || 'https://rickandmortyapi.com/api'
```

Comment:
TBH, I don't like env variables in front apps because someone can think that it's "configurable" when it's already deployed... (yes, I know that it sounds weird, but I've saw this misunderstanding before). Specially, in this case where api name is directly attached with value, makes no sense to have a env var, but it's only my POV

Reply:
So what do you suggest? Having there something like `http://localhost:300X` ?   I like to have there the env variable and put an example on `.env.example` file (or a comment) to explain what does that env variable do

Reply:
nope, we're not talking about `localhost:300x` it's for external APIs... whats de difference between having `.env` file with api url or directly to have it like `baseURL: 'https://rickandmortyapi.com/api'`?? And going further... could be more scalable to have a api "builder" passing the base url?
We'll take a look together ;-)

Reply:
but @jgleal , what if you want to test the frontend through your local backend?? Should we change the url of the api by hand??
I think it could be useful having a `.env` but we can discuss it further to understand better your point of view ;)